course/templates/course/index.html

```diff
@@ -8,89 +8,56 @@
 
 {% if welcome_text %}
 <div class="page-header">
-    <h1>{{ welcome_text|safe }}</h1>
+	<h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
-        <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
-                </div>
-                <div class="caption">
-                    <h3>{{ advert.title }}</h3>
-                    <p>{{ advert.text|safe }}</p>
-                    <div class="buttons caption">
-                        <a class="btn btn-default btn-block" role="button" href="{{ advert.href }}" target="_blank">
-                            <span class="glyphicon glyphicon-new-window" aria-hidden="true"></span>
-                            {% trans "Read more" %}
-                        </a>
-                    </div>
-                </div>
-            </div>
-        </div>
-    {% endif %}
+{% if my_instances %}
+	<div class="frontpage-panel">
+		<h2 class="panel-heading">{% trans "My courses"%}</h2>
+		<div class="row frontpage">
+			{% include "course/_course_cards.html" with instances=my_instances %}
+		</div>
+	</div>
+{% endif %}
+
```

Comment:
I think you can add a hard-coded default message here or somewhere around here. There will be no need to create a new issue on this.

Reply:
Alrighty, I will add those.

Reply:
These have now been added.